src/test/kotlin/com/atlassian/performance/tools/hardware/HwrDatasetCatalogue.kt

```diff
@@ -18,35 +17,25 @@ import java.time.Duration
 
 class HwrDatasetCatalogue {
 
-    fun xl7Mysql() = StorageLocation(
-        uri = URI("s3://jpt-custom-mysql-xl/dataset-7m-jira7"),
-        region = Regions.EU_WEST_1
-    ).let { location ->
-        Dataset(
-            label = "7M issues JSW 7 MySQL",
-            database = ConfigurableMysqlDatabase(
-                source = S3DatasetPackage(
-                    artifactName = "database.tar.bz2",
-                    location = location,
-                    unpackedPath = "database",
-                    downloadTimeout = Duration.ofMinutes(55)
-                ),
-                extraDockerArgs = listOf(
-                    "--max_connections=151",
-                    "--innodb-buffer-pool-size=40G",
-                    "--innodb-log-file-size=2146435072"
-                )
+    fun xl7Mysql() = Dataset(
+        label = "7M issues JSW 7 MySQL",
+        database = ConfigurableMysqlDatabase(
+            source = HttpDatasetPackage(
```

Comment:
Going for HTTP means it's gotta be public.
Did you check whether it contains a Jira license?